environment.py

```diff
@@ -230,9 +230,18 @@ def env() -> Mapping[str, Optional[str]]:
         #
         'AZUL_VERSIONED_BUCKET': 'edu-ucsc-gi-singlecell-azul-config-dev.{AWS_DEFAULT_REGION}',
 
-        # The number of workers pulling files from the repository (DSS or TDR).
+        # The number of workers pulling files from the DSS repository.
         # There is one such set of repository workers per index worker!
-        'AZUL_REPO_WORKERS': '8',
+        'AZUL_DSS_WORKERS': '8',
+
+        # The number of workers pulling files from the TDR repository.
+        # There is one such set of repository workers per index worker!
+        # We chose 1 TDR repository worker to avoid overloading BQ with
+        # excessive concurrent requests. Using 1 worker as opposed to 8 (or 4)
+        # improved noticeably the indexing time of TDR data. It also eliminated
+        # the errors caused by overloading BQ. Using 2 TDR workers performed
+        # approximately equal to using 1, so 1 worker was preferred.
```

Comment:
```suggestion
        # There is one such set of repository workers per index worker!
        # Using one worker as opposed to 8 (or 4)
        # improved the indexing time noticeably because it reduced
        # retries due to exceeding BigQuery's limit on the # of concurrent queries. Using two workers wasn't significantly faster.
```